app/autocomplete_mention/User.test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var expect = require('chai').expect;
+
+var database = require('../../lib/mongo')
+  , User = require('./User');
```

Comment:
The comma first pattern is different than the rest of the app. Try and keep the same style or add an eslint rule so that we can remain consistent.